src/demo-app/tsconfig.json

```diff
@@ -22,8 +22,11 @@
       ]
     }
   },
-  "exclude": [
-    "*-aot.ts"
+  "files": [
+    "./demo-app-types.d.ts",
+    "./demo-app-module.ts",
+    "./system-config.ts",
+    "./main.ts"
```

Comment:
Yeah, also I think that it's very typically for TS to not use a `glob` to walk through the files. Rather walking through the chain of files. Should we apply the merge ready label?